src/lib/error-messages.js

```diff
@@ -39,8 +39,9 @@ export const badBurstInput = errors => {
       propText = 'input';
   }
 
-  if (propText === propMap.idx)
+  if (propText === propMap.idx) {
```

Comment:
I've noticed elsewhere in the code that for single return if cases, you don't need to wrap the logic in curly braces, i.e.

```
if (propText === propMap.idx)
  return `Your ${propText} must be a positive integer.`;
```